docker/unittest-dedb/setup-dev-database.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -x
+set -e
+
+#psql -h 127.0.0.1 -U $POSTGRES_USER -d postgres -c "CREATE DATABASE de WITH OWNER $POSTGRES_USER;"
```

Comment:
I'm not sure it would be worth the effort to change this. The POSTGRES_USER is set in the Dockerfile and the setting is really only used when the image is built. You can override the POSTGRES_USER when firing up a container based on the image and you'll still have to connect to the database with the 'de' user. The POSTGRES_USER value is set in the Dockerfile.